docs/language/learn-ql/cpp/basic-query-cpp.rst

```diff
@@ -57,13 +57,13 @@ Running the query
 
    The query will take a few moments to return results. When the query completes, the results are displayed below the project name. The query results are listed in two columns, corresponding to the two expressions in the ``select`` clause of the query. The first column corresponds to the expression ``ifstmt`` and is linked to the location in the source code of the project where ``ifstmt`` occurs. The second column is the alert message.
 
-   ➤ `Example query results <https://lgtm.com/query/1505896936425/>`__
+   ➤ `Example query results <https://lgtm.com/query/4242591143131494898/>`__
 
    .. pull-quote::
 
       Note
 
-      An arrow (˅) at the bottom of the table indicates that more results are available. Click the arrow to display more results.
+      An ellipsis (…) at the bottom of the table indicates that the entire list is not displayed–click it to show more results.
```

Comment:
> not displayed–click it

Should there be spaces between the dash and the words?

Reply:
Different people have different styles, so we use the Microsoft Style Guide for consistency! Their style doesn't include space around the dash: https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/emes 👀